.travis.yml

```diff
@@ -13,10 +13,10 @@ before_script:
   - cd build
 
 script:
-  - ../configure CFLAGS="--coverage -ftest-coverage -fprofile-arcs" LDFLAGS=-lgcov --enable-debug && build-wrapper-linux-x86-64 --out-dir bw-output make all
+  - ../configure CFLAGS="--coverage -ftest-coverage -fprofile-arcs" LDFLAGS=-lgcov --enable-debug && build-wrapper-linux-x86-64 --out-dir bw-output make all || true  # Will always fail on external PRs.
   - make -k check VERBOSE=true || { test "$SEND_COVERAGE_EVEN_IF_TESTS_FAIL" = yes && curl -s https://codecov.io/bash > cov.sh && bash cov.sh -x "$GCOV"; }
-  - bash ../tests/recursively_generate_gcov.sh .
-  - cd .. && sonar-scanner
+  - bash ../tests/recursively_generate_gcov.sh . || true  # Failiures that occur here are harmless
+  - cd .. && sonar-scanner || true  # Will always fail on external PRs.
```

Comment:
since we only have external PRs, does it make sense to even run this? I assume you mean forks.

Reply:
It makes sense, as merge also triggers a build, which won't fail.